README.md

```diff
@@ -117,6 +116,39 @@ jobs:
           docker build . # will run buildx
 ```
 
+### Configure a registry mirror
```

Comment:
https://github.com/tonistiigi/binfmt/blob/master/.github/buildkit.toml is a somewhat nice pattern as well. Maybe we should show it as well.

Reply:
Good idea, done.

Reply:
Actually, I thought that sometimes I would prefer putting a file under `.github` to trying to inline toml inside yaml. Maybe intro "providing a toml configuration with a file or inline block" and leave one example to use one and second another syntax.

Reply:
Done